src/devtools/client/inspector/markup/actions/markup.ts

```diff
@@ -211,15 +210,14 @@ export function addChildren(
   return async (dispatch, getState, { ThreadFront, replayClient, protocolClient }) => {
     let childrenToAdd = childNodes;
 
-    if (!features.showWhitespaceNodes) {
-      childrenToAdd = childNodes.filter(nodeObject => {
-        const node = nodeObject?.preview?.node;
-        if (!node) {
-          return false;
-        }
-        return node.nodeType !== NodeConstants.TEXT_NODE || /[^\s]/.exec(node.nodeValue!);
-      });
-    }
+    // Filter out whitespace nodes (formerly a pref option)
```

Comment:
Curious why you removed the `showWhitespaceNodes` check?

Reply:
It was a pref that was never modified and its default value was set to `false`, so this check was always filtering them out anyway.